monsieurbiz/sylius-settings-plugin/1.0/config/routes/monsieurbiz_settings_plugin.yaml

```diff
@@ -0,0 +1,3 @@
+monsieurbiz_sylius_settings_admin:
+    resource: "@MonsieurBizSyliusSettingsPlugin/Resources/config/routes/admin.yaml"
+    prefix: /%sylius_admin.path_name%
```

Comment:
Will this really work? 
Excuse my ignorance, but I was sure you could not add parameters like this in the routes.

Reply:
It works pretty well indeed. We have the same in production.

The parameter is defined by Sylius itself here:

https://github.com/Sylius/Sylius/blob/f27be445c11eccd0556d5edbafd60067960ca46c/src/Sylius/Bundle/AdminBundle/Resources/config/app/config.yml#L38

This is actually very convenient to secure the admin path with a random name!

Reply:
@Nyholm What do you think?

Reply:
ping @Nyholm ? :p